src/Protocol/SymbolInformation.php

```diff
@@ -91,6 +90,14 @@ public static function fromNode(Node $node, string $fqn = null)
         }
         if ($node instanceof Node\Name) {
             $symbol->name = (string)$node;
+        } else if(
+            $node instanceof Node\Expr\FuncCall
+            && $node->name instanceof Node\Name
+            && strtolower((string)$node->name) === 'define'
+            && isset($node->args[0])
+            && $node->args[0]->value instanceof Node\Scalar\String_
+        ) {
+            $symbol->name = (string)$node->args[0]->value->value;
```

Comment:
In that line?
```php
 } else if (isset($node->name)) {
             $symbol->name = (string)$node->name;
} 
```

I'd debugged a bit and figured out that without this method it will be set to `define`(which is the name of the node). But we want the value as a name, so that's why.